WordPress/src/main/java/org/wordpress/android/util/wizard/WizardManager.kt

```diff
@@ -25,6 +25,19 @@ class WizardManager<T : WizardStep>(
     private fun isIndexValid(currentStepIndex: Int): Boolean {
         return currentStepIndex >= 0 && currentStepIndex < steps.size
     }
+
+    fun stepsCount(): Int = steps.size
+
+    /**
+     * Returns position of the step (starting from 1) or -1 if the step is not present.
+     */
+    fun stepPosition(T: WizardStep): Int {
+        return if (steps.contains(T)) {
+            steps.indexOf(T) + 1
+        } else {
+            -1
```

Comment:
Can we throw an exception when this happens or at least return `null` if it's a valid state we need to handle in runtime? If the concern here is that Java won't warn us for the `null` value, it's the case for `-1` as well. Could you elaborate on this decision?

Reply:
I'm not sure what I was thinking about. I guess I'm just used to return -1 when working with indices. Fixed in [270e831](https://github.com/wordpress-mobile/WordPress-Android/pull/8756/commits/270e831bf195de50a3f826ac19b6524fe044b85e)